index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="UTF-8">
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <nav class="menu">
+            <div class="logo"><h2>My Logo</h2></div>
+
+            <ul>
+                <li><a href="#" class="active" onclick="markMenuItem(this)">Portfolio</a></li>
+                <li><a href="#" onclick="markMenuItem(this)">About</a></li>
+                <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
+            </ul>
+
+            <div id="menuBtn" class="active">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+
+        <main>
+            <section class="info">
+                <h1>I’m Mariane Glad to see you!</h1>
+                <span class="infos">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</span>
+                <p>LET’S CONNECT</p>
+                <div class="socialMedia">
+                    <a href=""> <img src="assets/twitter.svg" alt=""></a>
+                    <a href=""> <img src="assets/linkedin.svg" alt=""></a>
+                    <a href=""> <img src="assets/mail.svg" alt=""></a>
+                    <a href=""> <img src="assets/github.svg" alt=""></a>
+                    <a href=""> <img src="assets/fingers.svg" alt=""></a>
+                </div>
+            </section>
```

Comment:
Kindly take note of the images on the page. There are some background images after the icons according to the Figma design.
**_Your Design 👇_**
<img width="378" alt="image" src="https://user-images.githubusercontent.com/73854999/204891103-915ecfe5-92e0-49e0-8ed1-5a71f296fb49.png">

**_Figma Design 👇_**
<img width="378" alt="image" src="https://user-images.githubusercontent.com/73854999/204889446-3a3d9d59-35e3-4d9d-b7f4-cdde5446e030.png">